.github/workflows/wheels.yml

```
@@ -175,3 +396,78 @@ jobs:
           files: dist/*.whl
         env:
             GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+  upload_artifactory:
+    name: Upload wheels to Artifactory
+    if: inputs.build_type == 'nightly' || inputs.build_type == 'release'
+    runs-on: { group: nvrgfx, labels: [Linux, X64] }
```

YAMLlint (1.38.0)

[error] too many spaces inside braces

(braces)